v-next/hardhat-node-test-reporter/src/error-formatting.ts

```diff
@@ -57,10 +68,9 @@ ${stack}`;
 
 // TODO: Do this in a more robust way and that works well with windows
 function replaceFileUrlsWithRelativePaths(stack: string): string {
-  return stack.replaceAll(
-    "(" + pathToFileURL(process.cwd() + "/").toString(),
-    "(",
-  );
+  return stack
+    .replaceAll("(" + pathToFileURL(process.cwd() + path.sep).toString(), "(")
+    .replaceAll("(" + process.cwd() + path.sep, "(");
```

Comment:
what's the difference with the previous replaceAll?

Reply:
See my previous comment